src/samplers/syscall/latency/mod.bpf.c

```diff
@@ -76,6 +116,22 @@ int sys_exit(struct trace_event_raw_sys_exit *args)
 		__sync_fetch_and_add(cnt, 1);
 	}
 
+	// for some syscalls, we track counts by "family" of syscall. check the
+	// lookup table and increment the appropriate counter
+	idx = 0;
+	if (syscall_id < 336) {
+		u32 counter_idx = syscall_lookup[syscall_id];
```

Comment:
We need a source for the 336 number and retrieval date, since it is likely to get updated in the future and also seems architecture dependent? I was looking at https://github.com/torvalds/linux/blob/master/arch/x86/entry/syscalls/syscall_64.tbl

Reply:
Yes, this will be arch dependent. I'm wondering if it's better to address that immediately, or merge and replace with a new strategy.

I'm considering having a map that the userspace writes into with the appropriate LUT for the runtime platform. This means that we can centralize the logic in userspace and keep the BPF a little more minimal. Also gives us some flexibility in terms of defining/enabling counts for groups by config if we want that in the future.

Reply:
I like this idea- put platform-specific initialization in one place before kicking off all the samplers.

 Do we have a global variable / switch for platform right now?

Reply:
I *think* the cfg stuff will get us most of the way there with target os and platform detection. I think it makes sense to start plumbing it in now.